### PR TITLE
Fixing `package_dir` with the new DCOS Cli

### PR DESCRIPTION
**Pull request by christopherhein**
**Why:**
- The latest version of DCOS removed the `package_dir` method
- This re-implements the `package_dir` method
- [modified private `_package_dir()` in the `dcos/subcommand.py`](https://github.com/mesosphere/dcos-cli/blob/master/dcos/subcommand.py#L251-L259)
- [Original public method in the `dcos/subcommand.py`](https://github.com/mesosphere/dcos-cli/blob/588428c2d1f022446d5907d35e4346a3a971bf1d/dcos/subcommand.py#L293-L302)

### CODE REVIEW

File: dcos_kubectl/cli.py [hunk 1]
```diff
@@ -146,17 +153,17 @@ def main():
...
         "--username=dcos-kubectl"  # to avoid username prompt
     ] + args, env=env)
-    sys.exit(rc)
+    sys.exit(ret_code)
 
 
 if __name__ == "__main__":
-    cfg = os.path.join(os.getenv("HOME"), dcos.constants.DCOS_DIR, "dcos.toml")
-    os.environ[dcos.constants.DCOS_CONFIG_ENV] = cfg
+    CFG = os.path.join(os.getenv("HOME"), constants.DCOS_DIR, "dcos.toml")
```
Comment (sttts) [inline dcos_kubectl/cli.py]: can we use the same "HOME logic" as above?

File: dcos_kubectl/cli.py [hunk 2]
```diff
@@ -38,6 +36,16 @@ def kubectl_binary_path_and_url(master, verify=True):
     return file_path, url
 
 
+def _package_dir(name):
+    dcos_dir = constants.DCOS_DIR
+    dcos_subcommand_dir = constants.DCOS_SUBCOMMAND_SUBDIR
+    subcommand_dir = os.path.expanduser(os.path.join("~",
```
Comment (sttts) [inline dcos_kubectl/cli.py]: ugly indention

File: dcos_kubectl/cli.py [hunk 3]
```diff
@@ -38,6 +36,16 @@ def kubectl_binary_path_and_url(master, verify=True):
...
+                                                     dcos_dir,
+                                                     dcos_subcommand_dir))
+    return os.path.join(subcommand_dir,
```
Comment (sttts) [inline dcos_kubectl/cli.py]: ugly indention

File: dcos_kubectl/cli.py (hunk 1 above)
Comment (sttts) [inline dcos_kubectl/cli.py]: Why is CFG in capital letters?

**Comment by sttts** [comment]
Some style nits. Otherwise, lgtm.

**Comment by christopherhein** [comment]
@sttts thanks for taking a look at this, made most of the adjustments, let me know if any of the indentation stuff is still and issue. I use [`pylint`](https://www.pylint.org/) in vim to keep things formatted, so thats where some of the changes had come from… 

As well I took your advise and refactored the `_dcos_dir` mechanisms out to use in the entrypoint method

**Comment by sttts** [comment]
lgtm.

**Pull request merged**